specs/referrer-policy/index.src.html

```diff
@@ -539,15 +539,15 @@ <h3 id="referrer-policy-delivery-referrer-attribute">Delivery
     <dd><a><code>Unsafe URL</code></a></dd>
   </dl>
 
-  A policy delivered via a <code>referrer</code> attribute on an element
+  A policy delivered via a <code>referrerpolicy</code> attribute on an element
   takes precedence over the policy defined for the whole document via CSP or
   a <a element>meta</a> element.
 
   ISSUE: If an <code><a element>a</a></code>
   or <code><a element>area</a></code> element includes both
-  a <code>referrer</code> attribute as well as
+  a <code>referrerpolicy</code> attribute as well as
   a <a href="https://html.spec.whatwg.org/#link-type-noreferrer"><code>noreferrer</code>
-  link type</a>, should the policy of the <code>referrer</code> attribute
+  link type</a>, should the policy of the <code>referrerpolicy</code> attribute
```

Comment:
@jeisinger done, taking the conservative approach: https://github.com/sc0ttbeardsley/webappsec/commit/67018e17b0cf52a3decb609a2105282f7a7817ae